diffkemp/simpll/ModuleComparator.cpp

```diff
@@ -62,7 +65,8 @@ void ModuleComparator::compareFunctions(Function *FirstFun,
                                          showAsmDiffs, &GS, DI, this);
     if (fComp.compare() == 0) {
         DEBUG_WITH_TYPE(DEBUG_SIMPLL,
-                        dbgs() << "Function " << FirstFun->getName()
+                        dbgs() << getIndent() << "Function "
```

Comment:
I think this should be indented with the same level as the "Comparing..." message (for example by setting the -1 indent before the print), not not indented at all.

Reply:
Yes, that is what I wanted to say.

Reply:
I have updated the `getDebugIndent` function to support indentation level offsetting for this use case.

Reply:
Is it necessary to change the `getDebugIndent` function? I think it would be better to decrease the indent level before.The same can be done for the message that the functions are not equal (actually, it can be decreased just once, right after running the `DifferentialFunctionComparator`). Then, when there is inlining done, the indent should be increased again.

Reply:
This is definitely correct. However, I feel like this use case scenario would occur more times in the future and it's simpler to put the solution to it directly inside the `getDebugIndent` function. I also think that this approach makes the indentation maybe a little less readable though... Should I remove the offset functionality then, or keep it?

Reply:
I agree with the fact that this approach makes the indentation less readable, which I think is the most important argument. Hence, I'd suggest to remove the offset functionality and require the user to explicitly increase or decrease the indentation level.
Looking for opinion from @lenticularis39 .

Reply:
It seems logical to me to explicitly decrease the indentation level at least in this case, because if I understand it correctly there is no additional decrease, it is only moved before the "Functions are equal" print.

Reply:
Alright, agreed. Will be removing the offset option.